src/reducers/fill-color-2.js

```diff
@@ -24,13 +24,12 @@ const reducer = function (state, action) {
             return state;
         }
         const colors = getColorsFromSelection(action.selectedItems);
-        if (colors.gradientType === GradientTypes.SOLID) {
-            // Gradient type may be solid when multiple gradient types are selected.
-            // In this case, changing the first color should not change the second color.
-            if (colors.fillColor2 === MIXED) return MIXED;
-            return state;
+        // Gradient type may be solid when multiple gradient types are selected.
+        // In this case, changing the first color should not change the second color.
+        if (colors.gradientType !== GradientTypes.SOLID || colors.fillColor2 === MIXED) {
```

Comment:
I think we figured out that the getColorsFromSelection function returns GradientType = SOLID, fill colors 1 and 2 = MIXED to indicate that the gradient type isn't actually solid, you've selected a bunch of things with different gradient types. (Just clarifying this for myself)